pko/helm/templates/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: package-operator-metrics
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: package-operator
+    port: metrics
```

Comment:
nit: Is this additional label and label selector in the ServiceMonitor needed since the ServiceMonitor already looks for an endpoint port named metrics?